src/pykeen/models/unimodal/structured_embedding.py

```diff
@@ -45,6 +55,11 @@ def __init__(
         random_seed: Optional[int] = None,
         regularizer: Optional[Regularizer] = None,
     ) -> None:
+        r"""Initialize SE.
+
+        :param embedding_dim: The entity embedding dimension $d$. Is usually $d \in [50, 300]$.
```

Comment:
This default range may only hold for some datasets. It might be better placed in the hpo_defaults

Reply:
Just trying to give some insight to end users. The documentation should guide the user to try something reasonable if they don't know what to do first. If this doesn't work, then they can try something else, but taking away suggestions because they're not always technically the best makes this a package people won't be able to use

Reply:
If you want to update it with more insight for the users, then that would be much appreciated! In fact, having useful hints in the documentation of a package is worth 1000x what we did in any of our papers.